catalogue/webapp/components/ImageEndpointSearchResults/ImageEndpointSearchResults.tsx

```diff
@@ -80,57 +80,53 @@ const ImageEndpointSearchResults: FunctionComponent<Props> = ({
   const { isFullSupportBrowser } = useContext(AppContext);
   const [expandedImage, setExpandedImage] = useState<Image | undefined>();
   const [isActive, setIsActive] = useState(false);
-  const fetching = useRef(false);
   const router = useRouter();
+  const toggles = useToggles();
 
-  const getImage = async routerImageId => {
-    fetching.current = true;
-    const apiUrl = `https://api.wellcomecollection.org/catalogue/v2/images/${routerImageId}`;
-    const image: Image = await fetch(apiUrl).then(res => res.json());
-
-    if (image) {
-      setExpandedImage(image);
-      setIsActive(true);
-    }
-    fetching.current = false;
-  };
-
-  const encodeDecodeImageIdInUrl = (mode: 'encode' | 'decode') => {
-    const searchTest = window.location.search;
-    const search = searchTest?.slice(1) || '';
-    const params = parse(search);
-
-    if (mode === 'encode') {
-      params.imageId = expandedImage?.id;
-    }
-    if (mode === 'decode') {
-      delete params.imageId;
-    }
+  const imageMap: Record<string, Image> = images.reduce(
+    (a, image) => ({ ...a, [image.id]: image }),
+    {}
+  );
 
-    router.push(`${router.pathname}?${stringify(params)}`, undefined, {
-      shallow: true,
-    });
+  const setImageIdInURL = (id: string) => {
+    window.location.hash = id;
```

Comment:
correct, it seems that nextjs went even deeper into being a backend framework, and since fragments do not get sent to servers, next removed the router event